src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -915,69 +915,67 @@
                 {
                     "type": "bool",
                     "name": "shaderSignedZeroInfNanPreserveFloat64",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormPreserveFloat16",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormPreserveFloat32",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormPreserveFloat64",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat16",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat32",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat64",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat16",
-                    "value": false,
-                    "comment": ["ROADMAP2024 but no good support yet"]
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat32",
-                    "value": false,
-                    "comment": ["ROADMAP2024 but no good support yet"]
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat64",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat16",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat32",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat64",
-                    "value": false
+                    "value": true
                 }
```

Comment:
coordinate with @nipunG314 if these need to be expose DISABLE (like the above) or expose REQUIRE

Reply:
I'd REQUIRE, it also means you need to always enable the shader float control vk1.2 feature (and fail if its not present) during device list and init